packages/clerk-js/src/ui/foundations/typography.ts

```diff
@@ -20,8 +20,8 @@ const letterSpacings = Object.freeze({
 // This is directly related to the createFontSizeScale function in the theme
 // ref: src/ui/customizables/parseVariables.ts
 const fontSizes = Object.freeze({
-  xs: '0.65rem',
-  sm: '0.73125rem',
+  xs: '0.6875rem',
+  sm: '0.75rem',
```

Comment:
We need to keep these numbers as is. These were carefully selected to be as close as possible to the font-sizes we want, while also keep the correct ratio between all font-sizes.

Reply:
Reverted it, thanks for catching it.

Some font sizes in our design are slightly different than those, so I'll leave it as is for the time being to avoid adding many custom sizes.
At some point, it'd be good for us to align both codebase and design sizes.